tutorials/workflow/csharp/combined-patterns/WorkflowApp/Activities/CheckShippingDestination.cs

```diff
@@ -0,0 +1,31 @@
+using System.Net;
+using Dapr.Workflow;
+
+namespace WorkflowApp.Activities;
+
+internal sealed class CheckShippingDestination : WorkflowActivity<Order, ActivityResult>
+{
+    private readonly HttpClient _httpClient;
+
+    public CheckShippingDestination(HttpClient httpClient)
+    {
+        _httpClient = httpClient;
+    }
+
+    public override async Task<ActivityResult> RunAsync(WorkflowActivityContext context, Order order)
+    {
+        Console.WriteLine($"{nameof(CheckShippingDestination)}: Received input: {order}.");
+
+        var response = await _httpClient.PostAsJsonAsync("/checkDestination", order);
+        if (response.StatusCode != HttpStatusCode.OK)
```

Comment:
There's an easier [helper method](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpresponsemessage.issuccessstatuscode?view=net-9.0#system-net-http-httpresponsemessage-issuccessstatuscode) `response.IsSuccessStatusCode` that covers even if we get a 204 or other 2XX status codes other than `200 OK`.